indra/newview/llviewershadermgr.cpp

```diff
@@ -2692,6 +2694,33 @@ BOOL LLViewerShaderMgr::loadShadersInterface()
 		success = success && gDebugProgram.createShader(NULL, NULL);
 	}
 
+	if (success)
+	{
+        for (S32 variant = 0; variant < NORMAL_DEBUG_SHADER_COUNT; ++variant)
+        {
+            LLGLSLShader& shader = gNormalDebugProgram[variant];
+            LLGLSLShader& skinned_shader = gSkinnedNormalDebugProgram[variant];
+            shader.mName = "Normal Debug Shader";
+            shader.mShaderFiles.clear();
+            shader.mShaderFiles.push_back(make_pair("interface/normaldebugV.glsl", GL_VERTEX_SHADER));
+            // *NOTE: Geometry shaders have a reputation for being slow.
+            // Consider using compute shaders instead, which have a reputation
+            // for being fast. This geometry shader in particular seems to run
+            // fine on my machine, but I won't vouch for this in
+            // performance-critical areas.  -Cosmic,2023-09-28
```

Comment:
Per @RunitaiLinden , geometry shaders do not have the best performance characteristics, but performance was tested to be reasonable for this debugging use case. I've added disclaimers accordingly.